sconstruct

```diff
@@ -61,6 +61,7 @@ keyCommandsLangBlacklist=set([])
 
 vars = Variables()
 vars.Add("version", "The version of this build", versionInfo.version)
+vars.Add("version_build", "A unique number for this build.", "0")

```

Comment:
Correct. versionBuild seems like bad English. buildVersion would be better. But since they are all specific to version I figured version_* was better.